### DISCUSSION TITLE
Acidisland World Generator new Terrain

### DISCUSSION THREAD

**Discussion by SurvivorsDev** [BG-Software-LLC/SSB-AcidIslands#24] [Q&A]
![image](https://github.com/BG-Software-LLC/SuperiorSkyblock2/assets/36204486/0eca9ffa-bd7f-483f-9175-c11de344fe78)

There needs to be live terrain for undersea and above sea. Normal Acisland has become commonplace now.

**Comment by Lemvs** [comment]
@OmerBenGera can you move this to the AcidIslands repository?
